From f7183d113341bcbd3442d544097babe0e1c810c9 Mon Sep 17 00:00:00 2001
From: Robert Kanter <rkanter@apache.org>
Date: Thu, 5 Feb 2015 14:20:28 -0800
Subject: [PATCH 0923/1023] MAPREDUCE-6233.
 org.apache.hadoop.mapreduce.TestLargeSort.testLargeSort
 failed in trunk (zxu via rkanter)

(cherry picked from commit e2ee2ff7d7ca429487d7e3883daedffbb269ebd4)

Change-Id: Ie36077249df80339b990c16c8e33727a4aa9b744
(cherry picked from commit fbc260da0eb6980433ad4096bd2857d65395e471)
---
 .../org/apache/hadoop/mapreduce/TestLargeSort.java |    1 +
 1 file changed, 1 insertion(+)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/TestLargeSort.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/TestLargeSort.java
index ab99a2f..eb48d67 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/TestLargeSort.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapreduce/TestLargeSort.java
@@ -55,6 +55,7 @@ public void testLargeSort() throws Exception {
     int[] ioSortMbs = {128, 256, 1536};
     for (int ioSortMb : ioSortMbs) {
       Configuration conf = new Configuration(cluster.getConfig());
+      conf.setInt(MRJobConfig.MAP_MEMORY_MB, 2048);
       conf.setInt(MRJobConfig.IO_SORT_MB, ioSortMb);
       conf.setInt(LargeSorter.NUM_MAP_TASKS, 1);
       conf.setInt(LargeSorter.MBS_PER_MAP, ioSortMb);
-- 
1.7.9.5

